scripts/ddtest

```diff
@@ -12,7 +12,7 @@ fi
 # retry docker pull if fails
 for i in {1..3}; do docker-compose pull -q testrunner && break || sleep 3; done
 
-FULL_CMD="pip install -q --disable-pip-version-check riot==0.17.3 && $CMD"
+FULL_CMD="pip install -q --disable-pip-version-check git+https://github.com/DataDog/riot@fix/dev-package-dedicated-venv && $CMD"
```

Comment:
Is this install step required? What happens if we remove it? @emmettbutler The problem I had is that it was re-installing riot from PyPI even though the CI setup step was pulling from GitHub.